pkg/analysis/passes/provenance/provenance.go

```diff
@@ -0,0 +1,169 @@
+package provenance
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"os"
+	"os/exec"
+	"regexp"
+	"strings"
+
+	"github.com/grafana/plugin-validator/pkg/analysis"
+	"github.com/grafana/plugin-validator/pkg/logme"
+)
+
+var (
+	noProvenanceAttestation = &analysis.Rule{
+		Name:     "no-provenance-attestation",
+		Severity: analysis.Warning,
+	}
+	invalidProvenanceAttestation = &analysis.Rule{
+		Name:     "invalid-provenance-attestation",
+		Severity: analysis.Warning,
```

Comment:
Should this be an error instead? (are we setting it just as a warning for now so we don't block submissions)?

Reply:
the problem is that we don't know if a provenance was generated or not, we only know that is not valid.

if we make this an error we'll start failing all github-stored packages